src/model/include/iDynTree/ModelTestUtils.h

```diff
@@ -8,15 +8,21 @@
 #include <iDynTree/Model.h>
 #include <iDynTree/FixedJoint.h>
 #include <iDynTree/RevoluteJoint.h>
+#include <iDynTree/RevoluteSO2Joint.h>
 #include <iDynTree/PrismaticJoint.h>
 #include <iDynTree/FreeFloatingState.h>
 #include <iDynTree/LinkState.h>
 
 #include <iDynTree/TestUtils.h>
 
 #include <cassert>
+#include <cmath>
 #include "IJoint.h"
 
+#ifndef M_PI
+#define M_PI 3.14159265358979323846
+#endif
```

Comment:
This is unnecessary, on *nix platforms `M_PI` is defined by default, while on Windows platform, `_USE_MATH_DEFINES` should be defined to ensure that `M_PI` is defined.